dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/NettyFrameworkImpl.java

```diff
@@ -161,7 +147,7 @@ public void serverStopping() {
             // set up future to log channel stopped messages
             for (Channel channel : activeChannels) {
                 channel.closeFuture().addListener(future -> {
-                    logChannelStopped(channel);
+                    stop(channel);
```

Comment:
logChannelStopped is still used elsewhere, is it too early here or why?

Reply:
I believe in this case this was a mistake that was corrected since calling only logChannelStopped doesn't actually stop the channel only logs it which means we could still have references to the channel still going. By calling stop all the references we keep are ensured to be cleared and it eventually also calls logChannelStopped so we should be good to go with the correction

Reply:
OK understood!